.github/workflows/documenter.yml

```diff
@@ -16,7 +16,7 @@ jobs:
         with:
           version: 1.4
       - name: Install dependencies
-        run: julia --project=docs/ -e 'using Pkg; Pkg.develop(PackageSpec(path=pwd())); Pkg.instantiate()'
+        run: julia --project=docs/ -e 'using Pkg; Pkg.instantiate(); Pkg.develop(PackageSpec(path=pwd()))'
```

Comment:
What's the reason for this?

Reply:
I didn't see this documented anywhere, but the original order does not work. It's apparently necessary to do things in this order when using an unregistered package in the docs.

Reply:
Yes that makes sense. In the previous order, `Pkg` would not use the `Manifest.toml` and install the latest release of `Zygote`.